docs/backend/configuration.md

```diff
@@ -40,3 +41,43 @@ object and viewed / modified via `/internal/config` (see
 [Accessing the admin panel](./admin.md)).
 
 ![Screenshot of site configuration admin interface](https://user-images.githubusercontent.com/47985/73627238-6276d500-467e-11ea-8724-afb703f056bc.png)
+
+## Vault
+
+The vault Ruby gem allows us to interact with
+[HashiCorp's Vault](https://www.vaultproject.io/docs/what-is-vault). In a
+nutshell, Vault is a tool for securely storing and accessing secrets. The way we
+have it configured, it is completely optional for running a Forem. To access it
+we use the wrapper `AppSecrets`.
+
+```ruby
+class AppSecrets
+  def self.[](key)
+    result = Vault.kv(namespace).read(key)&.data&.fetch(:value) if ENV["VAULT_TOKEN"].present?
+    result ||= ApplicationConfig[key]
+
+    result
+  rescue Vault::VaultError
+    ApplicationConfig[key]
+  end
+
+  def self.[]=(key, value)
+    Vault.kv(namespace).write(key, value: value)
+  end
+
+  def self.namespace
+    ENV["VAULT_SECRET_NAMESPACE"]
+  end
+  private_class_method :namespace
+end
+```
+
+As you can see in the code, we attempt to access a secret from Vault if it is
+enabled, ie the `VAULT_TOKEN` is present. If Vault is not enabled or we cannot
+find the secret in it, then we fallback on fetching the secret from the
+`ApplicationConfig`.
+
+One advantage to using Vault with Forem is that it allows you to update your
+secrets easily through the application rather than having to mess with ENV
+files. If you would like to tryout Vault follow our
+[installation guide for setting it up locally](installation/vault).
```

Comment:
```suggestion
[installation guide for setting it up locally](../installation/vault.md).
```
Does this path actually link correctly without a `../` before it? I'd suspect you need to change it to this since `/installation` and `backend` are in separate directories.

Reply:
Yeah I am pretty sure this will work. If you look at the installation README we have similar links
```
## Installing locally

You can install DEV to your local machine and we have instructions for
[Mac](/installation/mac), [Windows](/installation/windows) and
[Linux](/installation/linux).
```